src/core/Action.js

```diff
@@ -20,21 +19,21 @@ import {ACTION} from '../constants';
  * @extends BaseNode
  **/
 
-export default Class(BaseNode, {
-
-  /**
-   * Node category. Default to `ACTION`.
-   * @property {String} category
-   * @readonly
-   **/
-  category: ACTION,
+export default class Action extends BaseNode {
 
   /**
    * Initialization method.
    * @method initialize
    * @constructor
    **/
-  initialize: function(params) {
-    BaseNode.prototype.initialize.call(this);
+  constructor() {
+    super();
   }
```

Comment:
If I'm not mistaken, this can be omitted because the constructor for the parent class is called anyway.